media-ranker/app/controllers/application_controller.rb

```diff
@@ -1,5 +1,10 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  def index
+    @books = Book.order(rank: :desc).limit(3)
+    @albums = Album.order(rank: :desc).limit(3)
+    @movies = Movie.order(rank: :desc).limit(3)
```

Comment:
This `3` is a "magic number" -- a variable determined more or less arbitrarily and could easily change in the future. Suddenly the client wants the top 5! If this theme of 3 things was repeated elsewhere in the application it would be handy to store it as a constant so you only have to change it in one place.